package.json

```diff
@@ -126,11 +126,13 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-storybook": "^0.5.5",
     "execa": "^5.1.1",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
your package lock json file looks with to much changes. I think u run npm i without nvm use.

Reply:
i reverted to masters, then uses nvm use, then did npm i , this is the result. :(

Reply:
maybe due to npm modification?